static/js/activity.js

```diff
@@ -320,23 +321,86 @@ exports.insert_user_into_list = function (user_id) {
     compose_fade.update_one_user_row(elt);
 };
 
+function get_users(search_term) {
+    var all_persons = [];
+    if (search_term.indexOf('') !== -1) {
```

Comment:
This is an idiom I haven't seen before.  Why are you searching for an empty string?